<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpenseTracker</class>
 <widget class="QWidget" name="ExpenseTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExpenseTracker</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>331</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="headerLabel">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>EXPENSE MONTHLY BUDGET TRACKER</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Category:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="categoryBox"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Subcategory:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="subcategoryTextField"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Amount including tax:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="amountTextField"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateTextField"/>
    </item>
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Add Expense</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>313</width>
     <height>129</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Display your expenses by the following option:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="displayBox">
        <property name="placeholderText">
         <string>Select Option</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Display Graph by subcategories</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="displayBox_2">
        <property name="placeholderText">
         <string>Select Option</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="displayYear">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>-20</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Income Sources without Tax:</string>
   </property>
  </widget>
  <widget class="QLabel" name="displayTotalInc">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>40</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Income: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="displayTotalExp">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total expenses: 0</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>156</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Source:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sourceTextField"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Income:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="incomeTextField"/>
    </item>
    <item>
     <widget class="QPushButton" name="incomeButton">
      <property name="text">
       <string>Add Income</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="yearlyBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>300</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Select Option</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>280</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Display Yearly expense Report:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
